dev-docs/bidders/metax.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: bidder
+title: MetaX
+description: MetaX Prebid Bidder Adapter
+biddercode: metax
+tcfeu_supported: true
+gvl_id: 1301
+usp_supported: true
+coppa_supported: true
+gpp_supported: true
+schain_supported: true
+dchain_supported: false
+userIds: all
+media_types: banner, video, native, audio
+floors_supported: false
+pbjs: false
+pbs: true
+prebid_member: false
+multiformat_supported: will-bid-on-one
```

Comment:
safeframes_ok: true/false

is always a very helpful meta data to be available

Reply:
fixed